scripts/convert-bsp-specific-overrides

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+# Generalize overrides subsystem for NXP and Mainline support
```

Comment:
I guess this should say what the script does now, something like `# Convert old-style NXP overrides like :mx8 to new style BSP overides like :mx8-nxp-bsp`